bench/bench_eb.c

```diff
@@ -153,13 +153,13 @@ static void util(void) {
 }
 
 static void arith(void) {
-	eb_t p, q, r, t[EB_TABLE_MAX];
+	eb_t p, q, r, t[EB_RELIC_TABLE_MAX];
```

Comment:
It would seem more consistent I think. Just less to think about, if *all* RELIC symbols start with RELIC_. RELIC_EB_TABLE_MAX looks sensible to me. With that adjustment and the squashing mentioned by @dschuermann , this PR should be ready to merge. :)